### PR TITLE
[hi] Localize  Tightly Coupled Architectures in hindi.

### LINKED ISSUE #2640: [hi] Localize Tightly Coupled Architectures
Page content - https://glossary.cncf.io/tightly-coupled-architectures/

`Tightly Coupled Architectures.md` is not available in hindi .

# Please describe why this term needs to be updated
- `Tightly Coupled Architectures.md` term available in English only but This term also needs to be Localized in Hindi.
- visit on page- https://glossary.cncf.io/tightly-coupled-architectures/

### PR DESCRIPTION
**Pull request by Aarav238**
Signed-off-by : Aarav238 [aarav8090shukla@gmail.com](mailto:aarav8090shukla@gmail.com)

### Describe your changes
I have localized the ` Tightly Coupled Architectures ` in hindi.

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by kumarankit999 — COMMENTED** [review]
Hey @Aarav238, Thanks for raising this PR, but check the above error!

**Comment by kumarankit999** [comment]
Netlify build has been failed!

**Comment by Aarav238** [comment]
@kumarankit999 How I can resolve it ?

**Comment by Aarav238** [comment]
I have resolved them @kumarankit999

**Review by abhay-raj19 — APPROVED** [review]
LGTM 👍
Thanks for contributing @Aarav238

**Review by kumarankit999 — COMMENTED** [review]
Looking Fixed Now!!!

**Comment by Aarav238** [comment]
Please approve this @kumarankit999 .

**Review by kumarankit999 — APPROVED** [review]
Thanks for Contributing!!!

**Comment by kumarankit999** [comment]
@bishal7679 check this one PR!

File: content/hi/tightly-coupled-architecture.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: साइट विश्वसनीयता इंजीनियरिंग (Tightly Coupled Architecture)
```
Comment (Garima-Negi) [inline content/hi/tightly-coupled-architecture.md:2]: Hi can we please replace साइट विश्वसनीयता इंजीनियरिंग with टाइट्ली कपल्ड आर्किटेक्चर

File: content/hi/tightly-coupled-architecture.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+status: Completed
+category: वास्तुकला
+---
+
+
+कसकर युग्मित वास्तुकला एक वास्तुशिल्प शैली है जहां कई अनुप्रयोग घटक एक दूसरे पर निर्भर होते हैं (शिथिल रूप से युग्मित वास्तुकला के विपरीत प्रतिमान)। इसका मतलब यह है कि एक घटक में बदलाव से अन्य घटकों पर असर पड़ने की संभावना है। आम तौर पर अधिक शिथिल युग्मित वास्तुशिल्प शैलियों की तुलना में इसे लागू करना आसान होता है, लेकिन यह सिस्टम को कैस्केडिंग विफलताओं के प्रति अधिक संवेदनशील बना सकता है। उन्हें घटकों के समन्वित रोलआउट की भी आवश्यकता होती है जो डेवलपर उत्पादकता पर दबाव बन सकता है।
```
Comment (Garima-Negi) [inline content/hi/tightly-coupled-architecture.md:8]: Similarly here, please replace with exact words rather than rarely used words.

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

File: content/hi/tightly-coupled-architecture.md (hunk 2 above)
Comment (ankur0904) [inline content/hi/tightly-coupled-architecture.md:8]:
```suggestion
टाइटली कपल आर्किटेक्चर एक आर्किटेक्चर शैली है जहां कई अनुप्रयोग घटक एक दूसरे पर निर्भर होते हैं (लूसली कपल आर्किटेक्चर के विपरीत प्रतिमान)। इसका मतलब यह है कि एक घटक में बदलाव से अन्य घटकों पर असर पड़ने की संभावना है। आम तौर पर अधिक लूसली कपल आर्किटेक्चर की तुलना में इसे लागू करना आसान होता है, लेकिन यह सिस्टम को कैस्केडिंग विफलताओं के प्रति अधिक संवेदनशील बना सकता है। उन्हें घटकों के समन्वित रोलआउट की भी आवश्यकता होती है जो डेवलपर उत्पादकता पर दबाव बन सकता है।
```

File: content/hi/tightly-coupled-architecture.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+कसकर युग्मित एप्लिकेशन आर्किटेक्चर अनुप्रयोगों के निर्माण का एक काफी पारंपरिक तरीका है। हालांकि जरूरी नहीं कि वे माइक्रोसर्विस विकास की सभी सर्वोत्तम प्रथाओं के अनुरूप हों, लेकिन वे विशिष्ट परिस्थितियों में उपयोगी हो सकते हैं। वे लागू करने में तेज़ और सरल होते हैं और मोनोलिथिक अनुप्रयोगों की तरह वे प्रारंभिक विकास चक्र को गति दे सकते हैं।
```
Comment (ankur0904) [inline content/hi/tightly-coupled-architecture.md:10]:
```suggestion
टाइटली कपल आर्किटेक्चर अनुप्रयोगों के निर्माण का एक काफी पारंपरिक तरीका है। हालांकि जरूरी नहीं कि वे माइक्रोसर्विस विकास की सभी सर्वोत्तम प्रथाओं के अनुरूप हों, लेकिन वे विशिष्ट परिस्थितियों में उपयोगी हो सकते हैं। वे लागू करने में तेज़ और सरल होते हैं और मोनोलिथिक अनुप्रयोगों की तरह वे प्रारंभिक विकास चक्र को गति दे सकते हैं।
```

**Review by ankur0904 — CHANGES_REQUESTED** [review]
Please have a look. `कसकर युग्मित वास्तुकला` doesn't make sense and same for others.

cc: @Garima-Negi

**Review by Garima-Negi — CHANGES_REQUESTED** [review]

**Comment by Aarav238** [comment]
I have replaced all the words with exact words. @Garima-Negi  @abhay-raj19 @kumarankit999 @bishal7679 Please review.

File: content/hi/tightly-coupled-architecture.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+title: टाइट्ली कपल्ड आर्किटेक्चर (Tightly Coupled Architecture)
+status: Completed
+category: वास्तुकला
+---
+
+
+टाइट्ली कपल्ड आर्किटेक्चर एक आर्किटेक्चर शैली है जहां कई अनुप्रयोग घटक एक दूसरे पर निर्भर होते हैं (लूसली कपल आर्किटेक्चर के विपरीत प्रतिमान)। इसका मतलब यह है कि एक घटक में बदलाव से अन्य घटकों पर असर पड़ने की संभावना है। आम तौर पर अधिक लूसली कपल आर्किटेक्चर की तुलना में इसे लागू करना आसान होता है, लेकिन यह सिस्टम को कैस्केडिंग विफलताओं के प्रति अधिक संवेदनशील बना सकता है। उन्हें घटकों के समन्वित रोलआउट की भी आवश्यकता होती है जो डेवलपर उत्पादकता पर दबाव बन सकता है।
```
Comment (Garima-Negi) [inline content/hi/tightly-coupled-architecture.md:8]:
```suggestion
टाइट्ली कपल्ड आर्किटेक्चर एक आर्किटेक्चर शैली है जहां कई अनुप्रयोग घटक एक दूसरे पर निर्भर होते हैं (लूज़ली कपल्ड आर्किटेक्चर के विपरीत प्रतिमान)। इसका मतलब यह है कि एक घटक में बदलाव से अन्य घटकों पर असर पड़ने की संभावना है। आम तौर पर अधिक लूज़ली कपल्ड आर्किटेक्चर की तुलना में इसे लागू करना आसान होता है, लेकिन यह सिस्टम को कैस्केडिंग विफलताओं के प्रति अधिक संवेदनशील बना सकता है। उन्हें घटकों के समन्वित रोलआउट की भी आवश्यकता होती है जो डेवलपर उत्पादकता पर दबाव बन सकता है।
```

**Review by Garima-Negi — APPROVED** [review]

**Comment by Aarav238** [comment]
Thanks @Garima-Negi .

**Comment by kumarankit999** [comment]
Thanks for contributing @Aarav238

**Pull request merged**